lib/pymedphys/_gamma/implementation/shell.py

```diff
@@ -97,8 +98,12 @@ def gamma_shell(
         The number of bytes of RAM available for use by this function. Defaults
         to 0.8 times your total RAM as determined by psutil.
     quiet : bool, optional
-        Used to quiet informational printing during function usage. Defaults to
-        False.
+        Deprecated but maintained for now for backwards compatibility.
+        `gamma_shell` now utilises the `logging` module. You can set
```

Comment:
```suggestion
        `pymedphys.gamma` now utilises the `logging` module. You can set
```

Reply:
And maybe add a link to the following?

https://docs.python.org/3/howto/logging.html